### PR TITLE
Create Checklist component and write markup & add icons to images/join-us

### PR DESCRIPTION
**Pull request by demirsergen**
- I found the icons that were used in this component from the current codeforchicago website (jekyll project). Followed the same folder structure -> created folder called join-us inside of images.
- I didn't think this wouldn't be a reusable component so I copied the content exactly.

### CODE REVIEW

File: src/components/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Link } from 'gatsby'
```
Comment (daleinen7) [inline src/components/Card.jsx:1]: Looks like this is code from the other PR. There's a couple things we can do to fix this.
1. We can revert this back to what it was before. You should be able to just copy and paste from the current code base. 
2. We can make sure to merge your other PR first then see what merge conflicts happen or see if this just resolves itself. 
Either way let's make sure these changes aren't on this PR just for consistency's sake.

File: src/components/Checklist.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import checkmark from '../images/join-us/checkmark.svg'
+import longArrowAltRight from '../images/join-us/long-arrow-alt-right.svg'
+
+const Checklist = () => {
+  return (
+    <section>
+      <h1 class="joinHeader">Ready To Join?</h1>
```
Comment (daleinen7) [inline src/components/Checklist.jsx:8]: Let's make this an `h3`.

File: src/components/Checklist.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      <p>Complete the pre-onboarding checklist</p>
+      <ul class="joinChecklist">
+        <li>
+          <img src={checkmark} alt="checkmark" class="joinCheckmark" />
+          <a href="https://codeforchicago.org/projects" target="_blank">
```
Comment (daleinen7) [inline src/components/Checklist.jsx:13]: Let's make this a Gatsby `Link`.

**Review by daleinen7 — CHANGES_REQUESTED** [review]
A few minor changes, and then there's another PR's code on this one.

File: src/components/Checklist.jsx [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
   return (
     <section>
-      <h1 class="joinHeader">Ready To Join?</h1>
+      <h3 class="joinHeader">Ready To Join?</h3>
       <p>Complete the pre-onboarding checklist</p>
       <ul class="joinChecklist">
         <li>
           <img src={checkmark} alt="checkmark" class="joinCheckmark" />
-          <a href="https://codeforchicago.org/projects" target="_blank">
+          <Link to="https://codeforchicago.org/projects" target="_blank">
```
Comment (daleinen7) [inline src/components/Checklist.jsx:14]: Since this will be an internal link we can just make the "to" prop to just be `/projects`.

**Review by daleinen7 — CHANGES_REQUESTED** [review]
One minor change and this will be ready to go.

File: src/components/Card.jsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import React from 'react'
```
Comment (daleinen7) [inline src/components/Card.jsx:1]: Still looks like the card component is on this PR. Let's figure out a way to remove this file from the PR. Let me know if you need help with this.

**Review by daleinen7 — CHANGES_REQUESTED** [review]
This looks good other than the Card component being a part of the PR.

**Review by demirsergen — COMMENTED** [review]
I just deleted the Card.jsx file from the changed files section. Hope this will fix it!

**Comment by daleinen7** [comment]
> I just deleted the Card.jsx file from the changed files section. Hope this will fix it!

Looks like that brings up a conflict with the existing card component. Maybe merge with main then see if that resolves the conflict.

**Pull request closed without merging**